store/transfer_processor.go

```diff
@@ -198,8 +186,16 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 		return err
 	}
 
-	// Insert balance if it's a subscribed account
+	// Insert balance and calculate diff to total balances
+	addrDiff := make(map[common.Address]map[common.Address]*big.Int)
+	allAddrs := append(addrs, newAddrs...)
```

Comment:
In `addrs` and `newAddrs`, they may have the same address. Can `getLatestBalances` avoid returning the multiple balances for a given address?

Reply:
yes. i added a test with dup addresses and it return unique addresses